experiments/golden-results/StratoX-summary.txt

```diff
@@ -204,11 +204,6 @@ Error message: ATTRIBUTE_MACHINE_ROUNDING unsupported
 Nkind: N_Attribute_Reference
 --
 Occurs: 8 times
-Calling function: Do_Object_Declaration_Full
```

Comment:
Why are there now only 7 of these?

Reply:
the missing Nkind was an attribute for 'Min'.It is now being processed correctly.